src/fu-util.c

```diff
@@ -4865,6 +4955,22 @@ main(int argc, char *argv[])
 			      /* TRANSLATORS: command description */
 			      _("Inhibit the system to prevent upgrades"),
 			      fu_util_inhibit);
+
+	fu_util_cmd_array_add(cmd_array,
+			      "repair",
```

Comment:
Can this be something to do with security for it's name?  `security-repair`?  Or `security-harden`?  Any other ideas?

Reply:
`security-harden` is better, I think.